### PR TITLE
add ai generated desc to description

### PR DESCRIPTION
**Pull request by PavelLinearB**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=ECWtDXa)

### CODE REVIEW

File: .cm/ask-ai.cm [hunk 1]
```diff
@@ -12,6 +12,8 @@ automations:
...
       - true
     run:
-      - action: add-comment@v1
+      - action: update-description@v1
         args:
-          comment: "{{ source | askAI("based on the given context, describe in plain english the changes introduced in this PR.", env.OPEN_AI_TOKEN) | prependTitle("gitStream Describe PR") | encode }}"
\ No newline at end of file
+          concat_mode: append
+          description: |
+            "{{ source | askAI("based on the given context, describe in plain english the changes introduced in this PR.", env.OPEN_AI_TOKEN) | prependTitle("✨ /:\ gitStream Describe PR ✨") | encode }}"
```
Comment (vim-zz) [inline .cm/ask-ai.cm:19]: I would add a line before saying something like `_the following was added by gitStream askAI plugin_`

Comment (vim-zz) [reply .cm/ask-ai.cm:19]: you say it is done automatically? I dont think so. Or do you mention it as a reference for such behavior?

Comment (PavelLinearB) [reply .cm/ask-ai.cm:19]: It's done by piping the output of the filter to `prependTitle` (another plugin that adds a prefix markdown title)

Comment (PavelLinearB) [inline .cm/ask-ai.cm:19]: per @MishaKav: `updateTitle` doesn't support encoded data, yet.
